omod-1.10/src/test/java/org/openmrs/module/attachments/rest/AttachmentController1_10Test.java

```diff
@@ -234,4 +235,28 @@ public void postAttachment_shouldUploadFile() throws Exception {
 			Assert.assertNull(obs.getEncounter());
 		}
 	}
+	
+	@Test(expected = IllegalRequestException.class)
+	public void postAttachment_shouldNotUploadFileAboveSizeLimit() throws Exception {
+		
+		String fileCaption = "Test file caption";
```

Comment:
Please move this under the `// Setup` comment as it's part of the test case setup.